shell/scripts/extension/publish

```diff
@@ -172,10 +181,12 @@ if [ "${GITHUB_BUILD}" == "false" ]; then
     exit 1
   fi
 
-  docker images > /dev/null
-  if [ $? -ne 0 ]; then
-    echo "docker is not running - this is required to build container images for the UI Plugins"
-    exit 1
+  if [ "${PODMAN_CONTAINER}" == "false" ]; then
+    docker images > /dev/null
+    if [ $? -ne 0 ]; then
+      echo "docker is not running - this is required to build container images for the UI Plugins"
```

Comment:
Change the message depending on if it's docker or podman?

Reply:
Podman is daemonless so we don't need to check if it is running.